src/component/RankingList.tsx

```
@@ -0,0 +1,106 @@
+          >
+            {rank.toLocaleString("ko-KR")}위
+          </div>
+          <Avatar className="h-20 w-20 rounded-full">
+            <AvatarImage src={avatarUrl} alt={username} />
+            <AvatarFallback className="bg-[#D5DBEC]">
+              <svg
+                className="h-16 w-16 text-muted-foreground"
+                fill="currentColor"
+                viewBox="0 0 24 24"
+              >
+                <path d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)